src/nspawn/nspawn.c

```diff
@@ -3397,10 +3397,7 @@ static int inner_child(
                 if (r < 0)
                         return log_error_errno(r, "personality() failed: %m");
 #endif
-        } else if (arg_architecture >= 0 && arg_architecture != native_architecture())
-                return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP),
-                                       "Selected architecture '%s' not supported locally, refusing.",
-                                       architecture_to_string(arg_architecture));
+        }
```

Comment:
hmm, please don't drop this entirely. instead keep a log_notice() message here, saying what's going on.

The thing is that the qemu-bin-fmt is massively incomplete (only a subset of syscalls + ioctls hooked up), and hence the fact that the code runs non-natively is really worth mentioning.